mod-finance/schemas/order_transaction_summary.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Order transactions summary",
+  "type": "object",
+  "properties": {
+    "id": {
```

Comment:
It seems to me that it will be more convenient to use the `purchaseOrderId` as the primary key. This will allow us to use the `PgUtils.getById` method instead of `PhUtils.get` with query. It will also allow us to use RMB to create a foreign key to link `order_transaction_summary` with `temporary_order_transactions`. @folio-org/acquisitions What do you think?

Reply:
Anyway, the story description was updated as you described. So the schema updated as it proposed